net-analyzer/goaccess/goaccess-1.1.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="A real-time web log analyzer and interactive viewer that runs in a terminal"
+HOMEPAGE="http://goaccess.io"
+SRC_URI="http://tar.goaccess.io/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="btree bzip2 debug geoip getline libressl tokyocabinet ssl unicode zlib"
+
+RDEPEND="sys-libs/ncurses:0=[unicode?]
+	geoip? ( dev-libs/geoip )
+	libressl? ( dev-libs/libressl )
+	!tokyocabinet? ( dev-libs/glib:2 )
+	tokyocabinet? (
+		dev-db/tokyocabinet[bzip2?,zlib?]
+		btree? (
+			bzip2? ( app-arch/bzip2 )
+			zlib? ( sys-libs/zlib )
+		)
+	)
+	ssl? ( dev-libs/openssl:0= )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+REQUIRED_USE="btree? ( tokyocabinet ) bzip2? ( btree ) ssl? ( !libressl ) zlib? ( btree )"
+
+src_configure() {
+	econf \
+		$(use_enable bzip2 bzip) \
+		$(use_enable zlib) \
+		$(use_enable debug) \
+		$(use_enable geoip) \
+		$(use_enable tokyocabinet tcb $(usex btree btree memhash)) \
+		$(use_enable unicode utf8) \
+		$(use_with getline) \
+		$(use_with $(usex ssl ssl libressl) openssl)
```

Comment:
Yes, I tested it, it worked for me.
But, since "libressl?" goes under "ssl?" in RDEPEND now, it is not needed anymore.
And we can use $(use_with ssl openssl), which ssl implementation (openssl or libressl) will be used is not important here.

> usually both openssl and libressl work with --with-ssl

https://goaccess.io/download - upstream uses "--with-openssl" flag